tests/integration/targets/aci_interface_policy_link_level/tasks/main.yml

```diff
@@ -138,6 +138,37 @@
       - cm_add_policy_again_no_descr.proposed.fabricHIfPol.attributes.name == nm_add_policy_again_no_descr.proposed.fabricHIfPol.attributes.name == 'ansible_test'
       - cm_add_policy_again_no_descr.sent == nm_add_policy_again_no_descr.sent == {}
       - cm_add_policy_again_no_descr.previous.0.fabricHIfPol.attributes.dn== nm_add_policy_again_no_descr.previous.0.fabricHIfPol.attributes.dn == cm_add_policy_again_no_descr.current.0.fabricHIfPol.attributes.dn == nm_add_policy_again_no_descr.current.0.fabricHIfPol.attributes.dn == 'uni/infra/hintfpol-ansible_test'
+  
+  # CHANGE SPEED ON LINK LEVEL POLICY
+  - name: Test each speed setting for Link Level Policy
+    block:
+      - name: Add Link Level Policy with various speeds (check mode)
+        cisco.aci.aci_interface_policy_link_level:
+          <<: *interface_policy_link_level_present
+          speed: "{{ item }}"
+        check_mode: true
+        loop: ["100M", "1G", "10G", "25G", "40G", "50G", "100G", "200G", "400G", "auto", "inherit"]
+        register: cm_add_policy_speed
```

Comment:
seems you are missing assert for the checkmode, could you add assert for proposed?

Reply:
I think i got them now, but cannot test

Reply:
"inherit" removed from check test, as is the default on ACI and is not changed .